designcompose/src/main/java/com/android/designcompose/DocServer.kt

```diff
@@ -66,7 +67,32 @@ internal class LiveDocSubscriptions(
     val subscribers: ArrayList<LiveDocSubscription> = ArrayList(),
 )
 
+/**
+ * Design doc status
+ *
+ * Publicly accessible status for a DesignDoc. Accessible via DesignSettings.designDocStatuses
```

Comment:
Should this be a new public API, or only available to tests? (I think there might be some annotations to mark something as accessible during a compilation for tests only). I would not want to see production logic get built on top of `isRendered` etc because it'll be fragile if the doc ID changes (we added the callback API for that particular use case).

Reply:
Hm, yeah, maybe I should make this test-only.

Is there a way to get an immutable reference to a DesignDoc?

Reply:
I think the `isRendered` flag is _probably_ redundant vs checking for the `docIdSemanticsKey`, so I'll just remove it.